types/lib/utils/templatize.d.ts

```diff
@@ -16,14 +16,22 @@ declare class TemplateInstanceBase extends
   Polymer.PropertyEffects(
   Polymer.Element) {
 
+  /**
+   * Find the parent model of this template instance.  The parent model
+   * is either another templatize instance that had option `parentModel: true`,
+   * or else the host element.
+   */
+  readonly parentModel: Polymer.PropertyEffects;
+  _methodHost: Polymer.PropertyEffects;
```

Comment:
It seems that we need to update our fields to make them `@private`. Extracted issue for that: https://github.com/Polymer/polymer/issues/5028